persistence/src/main/resources/reference.conf

```diff
@@ -5,7 +5,8 @@
 # This is the reference config file that contains all the default settings.
 # Make your edits in your application.conf in order to override these settings.
 
-# Directory of persistence journal and snapshot store plugins is available at the 
+# Directory of persistence journal and snapshot store plugins is available at the
+# TODO: Do we need to extend community url of pekko as well along with akka community?
```

Comment:
I don't know what to do, so left `TODO`

Reply:
please remove the TODO - the fact that the akka url is there will attract attention

it's ok to leave the https://akka.io/community/ as is until we work out what to replace it with